Automattic/wp-calypso
# Hosting Command Palette: Add command to navigate to Jetpack modules

Related to https://github.com/Automattic/dotcom-forge/issues/4859

## Proposed Changes

This PR adds a command to navigate to the Jetpack modules in `wp-admin`. This command is available either for Atomic sites or for self-hosted sites. 

<img width="991" alt="Screenshot 2023-12-21 at 3 57 29 PM" src="https://github.com/Automattic/wp-calypso/assets/25575134/7057110a-1545-4663-881a-f0de98045f5a">


## Testing Instructions

* Navigate to Calypso Live Link
* Press `cmd + k` on macOS or `ctrl + k` on Windows to open the command palette
* Search for the `Manage Jetpack modules` command
* Select the command
* Confirm that you can only see Atomic (including staging) or self-hosted sites in the list
* Select an Atomic site to navigate to and trigger the command
* Confirm that you are brought to Jetpack modules when executing the command: `/wp-admin/admin.php?page=jetpack_modules`


## Pre-merge Checklist
